<dec f='include/c++/12.2.0/bits/functexcept.h' l='109' type='void std::__throw_future_error(int )'/>
<doc f='include/c++/12.2.0/bits/functexcept.h' l='107'>// Helpers for exception objects in &lt;future&gt;</doc>
